DMX.Core.Api.Tests.Unit/Services/Orchestrations/LabOrchestrationServiceTests.Validations.RetrieveById.cs

```diff
@@ -0,0 +1,58 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using DMX.Core.Api.Models.Foundations.Labs;
+using DMX.Core.Api.Models.Foundations.Labs.Exceptions;
+using DMX.Core.Api.Models.Orchestrations.Labs.Exceptions;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace DMX.Core.Api.Tests.Unit.Services.Orchestrations
+{
+    public partial class LabOrchestrationServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowOrchestrationValidationExceptionOnRetrieveByIdIfLabIdIsInvalidAndLogItAsync()
+        {
+            // given
+            var invalidLabId = Guid.Empty;
+            var invalidLabException = new InvalidLabException();

```

Comment:
Every service owns it's own exceptions. so we should make an exception called `InvalidLabOrchestrationException`

Reply:
Feel free to fix this across all. I see this pattern in previous methods as well.